docs/standards/lsp-background/erc725.md

```diff
@@ -14,69 +13,78 @@ sidebar_position: 1
 
 ## Introduction
 
-Using Externally Owned Accounts (EOA) is limiting and risky as if the private key of the account gets leaked, anyone can execute any transaction from it, transferring all assets owned by the account. Also, data cannot be attached to it and it's limited to performing the CALL and CREATE opcode.
-
-In order to have a more secure account, more flexible execution on the blockchain and the ability to attach endless data, a smart contract should be used. It ensures a safe ownership process, more operations to execute and endless data store availability.
-
-
-## What does this standard represent?
+EOAs (Externally Owned Accounts) are controlled by private keys and can perform specific operations such as **CALL** to interact with other addresses and **CREATE** to create other smart contracts. However, they do not have any internal logic or storage capabilities.
```

Comment:
maybe you could put a link on the words "call" and "create"? To this pages or smtgh similar:

- https://ethereum.org/en/developers/docs/evm/opcodes/
- https://github.com/wolflo/evm-opcodes/blob/main/gas.md#a9-create-operations

I'm not sure if we did it somewhere else already